src/functions.rs

```diff
@@ -961,6 +961,75 @@ fn call_to_fsc_rules<'a>(
     Ok(ret)
 }
 
+fn call_to_sids<'a>(
+    c: &FuncCall,
+    types: &TypeMap,
+    class_perms: &ClassList,
+    context: &BlockContext<'_>,
+    file: &SimpleFile<String, String>,
+) -> Result<Vec<Sid<'a>>, CascadeErrors> {
+    if context.in_function_block() {
+        return Err(CascadeErrors::from(
+            ErrorItem::make_compile_or_internal_error(
+                "initial_context() calls are not valid in functions",
+                Some(file),
+                c.name.get_range(),
+                "Remove this call",
```

Comment:
add "or move to global scope"?

Reply:
I thought about that.  The problem is that the most idiomatic thing for a user to do would be to put it inside a type definition.  Global scope is fine of course, but I would expect the common case to be:

```
resource kernel_sid {
    initial_context("kernel", this);
}
```

Maybe this rewrite would be more helpful?  "You may want to place this call directly inside a type definition"

Reply:
I like that better.